src/index.scss

```diff
@@ -20,9 +21,3 @@ main {
     padding-right: 1rem;
   }
 }
-
-/// Padding was introduced in 10.3.0, needs an option to let grid set the viewport gutter
-/// @link https://github.com/carbon-design-system/carbon/issues/3010
-.bx--content {
-  padding: 0;
-}
```

Comment:
This class is needed. Otherwise there is too much padding in your layout: https://deploy-preview-1505--carbon-tutorial.netlify.com/#/